src/nbsap/views/indicators.py

```diff
@@ -38,7 +39,8 @@ def eu_indicators(request):
 
 @auth_required
 def list_eu_indicators(request):
-    indicators = models.EuIndicator.objects.filter(parents=None).all()
+    indicators = models.EuIndicator.objects.filter(
+        parents=None).all().order_by('title_' + settings.LANGUAGE_CODE)
```

Comment:
I think we should take the current language from the request, not the default one from settings.